src/vuln_analysis/utils/functions_parsers/lang_functions_parsers.py

```diff
@@ -121,7 +123,7 @@ def is_script_language():
         return False
 
     def is_same_package(self, package_name_from_input, package_name_from_tree):
-        return package_name_from_input.lower() in package_name_from_tree.lower()
```

Comment:
@tmihalac This older implementation should be implemented now in golang subclass ( `GoLanguageFunctionsParser`), until we'll do a more precise implementation for golang ( with prefix and suffix checks)